extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/dart/controller/sql/DartSqlEngine.java

```diff
@@ -70,6 +72,21 @@ public class DartSqlEngine implements SqlEngine
   private final DartControllerConfig controllerConfig;
   private final ExecutorService controllerExecutor;
 
+  @Inject
```

Comment:
Is this still a singleton after removing the `@LazySingleton`? If it's only getting injected into a single object, it will still be fine; I just want to make sure there is only a single `DartSqlEngine` ever. It's important because it owns the executor that controllers run in.